package.json

```diff
@@ -35,7 +35,8 @@
     "test:group:admin": "yarn jest --forceExit --testPathPattern 'src/admin/.*'",
     "test:group:producer": "yarn jest --forceExit --testPathPattern 'src/producer/.*'",
     "test:group:consumer": "yarn jest --forceExit --testPathPattern 'src/consumer/.*.spec.js'",
```

Comment:
@fbonecco-cribl so, were the network tests just not being run, then? :disappointed:

Reply:
`test:group:network` is a just a new alias to be able to run these network-related tests alone. These are also executed as part of the main suite.